packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/Providers.tsx

```diff
@@ -36,7 +37,9 @@ export function Providers({ children }: { children: ReactNode }) {
   return (
     <WagmiProvider config={wagmiConfig}>
       <QueryClientProvider client={queryClient}>
-        <RainbowKitProvider theme={darkTheme()}>{children}</RainbowKitProvider>
+        <RainbowKitProvider theme={darkTheme()}>
+          <WorldStoreProvider>{children}</WorldStoreProvider>
```

Comment:
is this necessary?

Reply:
Yeah, set it up using this guide https://zustand.docs.pmnd.rs/guides/nextjs it helps to prevent weird SSR bugs when the store is initiated twice, once on server-side, and next on client-side.

Reply:
yeah since this tooling is all client-only, I have a preference for this being simpler and check if mounted before using to avoid SSR errors

if you really insist on this approach, it might be nice to explore generalizing this into a `ZustandStoreProvider` or something

Reply:
Yeah I see your point, simplified it to a single `store.ts`.